jp.yml

```diff
@@ -0,0 +1,279 @@
+        homepage: "ホームページ"
+        github: "GitHub"
+        twitter: "Twitter"
+      processes:
+        all: "All processes"
+        allBut: "All but {{list}}"
+        except: "{{included}} except {{excluded}}"
+        tooltip:
+          target: "Target process"
+          targets: "Target processes:"
+          excluded: "Excluded:"
+      update: "アップデート"
+      updateNotNeeded: "インストールされたバージョンは、最新です"
+      install: "インストール"
+      compile: "コンパイル"
+      disable: "無効化"
+      enable: "有効化"
+      edit: "編集する"
+      fork: "フォークする"
+      remove: "削除"
+    details:
+      title: "詳細"
+      noData: "Modの詳細は不明。"
+    settings:
+      title: "設定"
+      noData: "利用できるmodの設定はありません。"
+      saveButton: "設定を保存"
+      sampleValue: "サンプル値"
+      arrayItemAdd: "新しい項目を追加"
+      arrayItemRemove: "項目を削除"
+    code:
+      title: "ソースコード"
+      noData: "Modのソースがありません。"
+      collapseExtra: "Readmeと設定を折りたたむ"
+    changelog:
+      title: "変更履歴"
+      loadingFailed: "変更履歴のロードに失敗しました。 <0>ここをクリック</0> してGitHubを開く"
+    advanced:
+      title: "詳細"
+      debugLogging:
+        title: "Debug logging"
+        description: "modのトラブルシューティングに役立ちます。"
+        none: "なし"
+        modLogs: "Mod logs"
+        detailedLogs: "Detailed debug logs"
+        showLogButton: "Show log output"
+      modSettings:
+        title: "Modの設定"
+        description: "modの設定を簡単にエクスポートまたは共有できます。"
+        loadButton: "ロード"
+        loadFormattedButton: "フォーマットされたものをロード"
+        saveButton: "保存"
+        invalidData: "無効なJSONデータ"
+      customList:
+        titleInclusion: "カスタムプロセス包含リスト"
+        descriptionInclusion: "modがターゲットとする追加の実行ファイル名/パスのカスタムリストです。このリストは、どのプロセスをターゲットにするかを決定するために、 modの作者が定義した包含リストに追加されます。各プロセスについて、実行ファイルパスがインクルードエントリの一つにマッチし、かつ、 除外エントリにマッチしない場合に、modがロードされます。"
+        titleExclusion: "カスタムプロセス除外リスト"
+        descriptionExclusion: "modがターゲットから除外する追加の実行ファイル名/パスのカスタムリストです。このリストは、どのプロセスをターゲットにするかを決定するために、 modの作者が定義した除外リストに追加されます。各プロセスについて、実行可能ファイルパスがインクルードエントリの一つにマッチし、除外エントリにマッチしない場合に、modがロードされます。"
+        processListPlaceholder: "プロセス名/パス、1行に1つずつ、 例:"
+        saveButton: "保存"
+    changes:
+      title: "最新バージョンの変更点"
+      noData: "インストールされたバージョンは最新です。"
+      splitView: "スプリット標示"
+      expandLines_one: "隠された一行を展開"
+      expandLines_other: "隠された{{count}}行を展開"
+  home:
+    browse: "Modをブラウズ"
+    installedMods:
+      title: "インストールされたMod"
+      noMods: "modはインストールされていません"
+    featuredMods:
+      title: "注目のMod"
+      noMods: "まだインストールされていない注目のmodはありません"
+      explore: "他のModを探す"
+  explore:
+    search:
+      placeholder: "mod検索中..."
+      popularAndTopRated: "人気で評価が高い"
+      popular: "人気"
+      topRated: "評価が高い"
+      newest: "最新"
+      lastUpdated: "最終更新日"
+      alphabeticalOrder: "アルファベット順"
+  settings:
+    language:
+      title: "言語"
+      description: "Windhawkのお好みの表示言語を選択してください。"
+      contribute: "<0>新しい翻訳に貢献する</0>."
+      credits: "<0>lmncaj</0>によって日本語翻訳されました。"
+      creditsLink: "mailto:john.smith@example.com"
```

Comment:
Please replace with your email or website, or, if you prefer not to have a link, remove the `<0>...</0>` tag from the `credits` string.